ext/console/02_console.js

```diff
@@ -347,6 +347,14 @@
       // use generic 'Function' instead.
       cstrName = "Function";
     }
+    let stringValue = "";
+    if (typeof value.toString === "function") {
+      stringValue = value.toString();
+    }
```

Comment:
Please use primordials

```suggestion
    const stringValue = FunctionPrototypeToString(value);
```

Reply:
Sounds good
Thanks for reminding me